### ISSUE TITLE
[9.3.3][BETA 9.4] Add an authorization to a user tab & profiles selection

### ISSUE THREAD

**Issue by seb1435** [glpi-project/glpi#5098]
**Describe the bug**

https://forum.glpi-project.org/viewtopic.php?pid=356072

Only users with Super-admin profile can select all the profiles available in the Authorization tab when editing an user.

Other profiles like Admin can only select Admin or Self-service

**To reproduce**

1. Add a user with Super-admin and Admin profiles
2. Log-in using this user and select Super-admin profile
3. Edit an user
4. Go on the Authorizations tab
5. Open the Profile dropdown: all available profile are listed
6. Select Admin profile
7. Open the Profile dropdown: only Admin and Self-service are available for selection

**Expected behavior**

As Admin is allowed to read profiles in Administration > Profiles > Administration, all profiles with lower rights should be available for selection in the dropdow list (as Technician or Supervisor).

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```
[code]
 
GLPI 9.3.3 ( => /var/www/glpi)
Installation mode: TARBALL
```

**Additional context**

Behavior also presents in beta 9.4

**Comment by OscarBeiro** [comment]
This issue is inherited.
Already present on 9.2.x

**Comment by cconard96** [comment]
I think the dropdown is working properly, but default profile rights are not correct (tested from 9.4).
Example, a Technician has read access to public RSS and public reminders, but an Admin does not. Another example is a Technician and Supervisor has access to read entities but an Admin does not.

Therefore, the Admin profile is less privileged than other profiles and cannot assign them to users. Other issues exist between other profiles.

**Comment by seb1435** [comment]
> Therefore, the Admin profile is less privileged than other profiles and cannot assign them to users. Other issues exist between other profiles.

So maybe the problem is the way GLPI calculates the rank of priviledge of a profile? For exemple, in my setup, Admin profile has more rights on configuration and  administration tabs than Supervisor but less on Assistance tab.

**Comment by cconard96** [comment]
The way I understand it, the admin (or which ever profile you use) has to have at the least the same level of access for every right. I assume this is to prevent an Admin from creating a more privileged profile and assigning it to themselves to gain more rights.

**Comment by seb1435** [comment]
Indeed, but GLPI should weight the differents rights, at least for administration and configuration rights don't you think ? A supervisor should be able to create another supervisor, a technician, etc.

**Comment by ConteZero** [comment]
@trasher @cconard96 current merged commit doesn't completely solve the bug.
I've tried to explain why these changes are not enough on this comment https://github.com/glpi-project/glpi/pull/5215#issuecomment-452636924

The following rights should be added to each default profile created from glpi-empty.sql:
rule_asset    0
global_validation    0

**Comment by trasher** [comment]
@ConteZero yes, I know. But current PR do provide fixes; another one can be opened for other issues.
